mycore-solr/src/main/resources/components/solr/config/mycore.properties

```diff
@@ -88,10 +73,8 @@ MCR.Solr.TikaFields=stream_size,stream_name,stream_source_info,stream_content_ty
 MCR.Solr.JoinQueryFields=%MCR.Solr.ContentBaseFields%,%MCR.Solr.AVExtenderFields%,%MCR.Solr.TikaFields%
 
 MCR.CLI.Classes.Internal=%MCR.CLI.Classes.Internal%,org.mycore.solr.commands.MCRSolrCommands
-MCR.URIResolver.xslImports.solr-document=%MCR.URIResolver.xslImports.solr-document%,mycoreobject-dynamicfields.xsl
-MCR.URIResolver.xslImports.solr-document-3=%MCR.URIResolver.xslImports.solr-document-3%,solr/indexing/mycoreobject-dynamicfields-3.xsl
-MCR.URIResolver.xslIncludes.xeditorTemplates=%MCR.URIResolver.xslIncludes.xeditorTemplates%,solr-xeditor-templates.xsl
-MCR.URIResolver.xslIncludes.xeditorTemplates-3=%MCR.URIResolver.xslIncludes.xeditorTemplates-3%,solr/xeditor/solr-xeditor-templates-3.xsl
+MCR.URIResolver.xslImports.solr-document=%MCR.URIResolver.xslImports.solr-document%,solr/indexing/mycoreobject-dynamicfields.xsl
+MCR.URIResolver.xslIncludes.xeditorTemplates=%MCR.URIResolver.xslIncludes.xeditorTemplates%,solr/xeditor/solr-xeditor-templates.xsl
```

Comment:
I thing, the path should also start with `/xsl` or `/xslt` to distinguish version 1 and version 3 XSL templates.
Or does that break to much existing code?

Reply:
as abvoe: xlsImport does not allow for adding resource:xsl/ or /xslt/ in the beginning. I would suggest to adapt these references in a separate PR after we added this.